Javascript.md

```diff
@@ -1,5 +0,0 @@
-# JavaScript: Basics and Tricky Questions
-
-### What are the differences between null and undefined?
-- `null`: No value. Null can be assigned to any variable but is not an object. typeof null returns `object`
-- `undefined`
```

Comment:
yeah okay. i'll probably try to split it up tomorrow or something. too lazy now HAHA